src/current/_includes/v22.2/orchestration/kubernetes-stop-cluster.md

```diff
@@ -7,37 +7,41 @@ To shut down the CockroachDB cluster:
     ~~~
 
-    This will delete the CockroachDB cluster being run by the Operator. It will *not* delete the persistent volumes that were attached to the pods. 
+    This will delete the CockroachDB cluster being run by the Operator. It intentionally does _not_ delete:
 
-    {{site.data.alerts.callout_danger}}
-    If you want to delete the persistent volumes and free up the storage used by CockroachDB, be sure you have a backup copy of your data. Data **cannot** be recovered once the persistent volumes are deleted. For more information, see the [Kubernetes documentation](https://kubernetes.io/docs/tasks/run-application/delete-stateful-set/#persistent-volumes).
-    {{site.data.alerts.end}}
+    - The persistent volumes that were attached to the pods, to avoid the risk of unintentional data loss. Before deleting a cluster's persistent volumes, be sure to back them up. For more information, refer to [Delete a Cluster's Persistent Volume](#delete-a-clusters-persistent-volumes) in the Kubernetes project's documentation.
+    - Any secrets you may have created. For more information on managing secrets, refer to the [Kubernetes project's documentation](https://kubernetes.io/docs/tasks/configmap-secret/managing-secret-using-kubectl).[Managing Secrets Using `kubectl`](https://kubernetes.io/docs/tasks/configmap-secret/managing-secret-using-kubectl) in the Kubernetes project's documentation.
 
-{{site.data.alerts.callout_info}}
-This does not delete any secrets you may have created. For more information on managing secrets, see the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configmap-secret/managing-secret-using-kubectl).
-{{site.data.alerts.end}}
 </section>
 
 <section class="filter-content" markdown="1" data-scope="manual">
-1. Delete the resources associated with the `cockroachdb` label, including the logs and Prometheus and Alertmanager resources:
 
-    {{site.data.alerts.callout_danger}}
-    This does not include deleting the persistent volumes that were attached to the pods. If you want to delete the persistent volumes and free up the storage used by CockroachDB, be sure you have a backup copy of your data. Data **cannot** be recovered once the persistent volumes are deleted. For more information, see the [Kubernetes documentation](https://kubernetes.io/docs/tasks/run-application/delete-stateful-set/#persistent-volumes).
-    {{site.data.alerts.end}}
+This procedure shuts down the CockroachDB cluster and deletes the resources you just created, including the logs and Prometheus and Alertmanager resources. This command intentionally does _not_ delete:
+
+- The persistent volumes that were attached to the pods, to avoid the risk of unintentional data loss. Before deleting a cluster's persistent volumes, be sure to back them up. For more information, refer to [Delete a Cluster's Persistent Volume](#delete-a-clusters-persistent-volumes) in the Kubernetes project's documentation.
```

Comment:
```suggestion
- The persistent volumes that were attached to the pods, to avoid the risk of data loss. Before deleting a cluster's persistent volumes, be sure to back them up. For more information, refer to [Delete a Cluster's Persistent Volume](#delete-a-clusters-persistent-volumes) in the Kubernetes project's documentation.
```

Reply:
I won't remove any more of the "unintentional" occurrences in case it's not something you want to update.